.example.env

```diff
@@ -17,4 +17,12 @@ REACT_APP_PINATA_GATEWAY_HOSTNAME=
 REACT_APP_BLOCKNATIVE_API_KEY=
 
 # sentry url. Not needed for development
-REACT_APP_SENTRY_DSN=
\ No newline at end of file
+REACT_APP_SENTRY_DSN=
+
+# Secret phrase used for e2e testing with metamask.
+# !!-- ** USE A TEST WALLET, NOT YOUR REAL WALLET!!!! ** -- !! 
+E2E_SECRET_WORDS=
```

Comment:
guess we'll need a shared test wallet for the GitHub action/ci?

Reply:
Yeah totally - probably want it to be privileged too if its possible to stop bad actors disrupting it.